### PR TITLE
[STAN-1055] if we have a dataset, use its title in the breadcrumb

### PR DESCRIPTION
**Pull request by roc**
### Before
Capture d’écran 2022-11-09 à 16 05 12

### After
Capture d’écran 2022-11-09 à 16 02 55

### CODE REVIEW

**Review by iebarat — APPROVED** [review]

**Review by bdsharma — APPROVED** [review]
Looks good

File: ui/components/Layout/index.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ export default function Home({ children, ...props }) {
 
   const { content } = useContentContext();
   const { title } = content;
+  const { title: datasetTitle } = props.data || false;
```
[the remark this answers is not in the document]
Comment (roc) [reply ui/components/Layout/index.js:41]: that's about it. Could do `{datasetTitle} = {...props.data}`?

[the remark this answers is not in the document]
Comment (lennym) [reply ui/components/Layout/index.js:41]:
```suggestion
  const datasetTitle = get(props, 'data.title');
```

[the remark this answers is not in the document]
Comment (roc) [reply ui/components/Layout/index.js:41]: Aight!

**Pull request merged**